plugins/async-node/ios/Tests/AsynNodePluginTests.swift

```diff
@@ -292,6 +292,147 @@ class AsyncNodePluginTests: XCTestCase {
         XCTAssert(count == 3)
         XCTAssertEqual(expectedMultiNode2Text, "new node from the hook 2")
     }
+
+    func handleEmptyNode() {
```

Comment:
this it possible to change this test to have similar logic to the android test https://github.com/player-ui/player/pull/488/files#diff-ac6259632cb11fb85e6116a391926ea88917bc5f23faba6c0657f014a09d74c0R320?

Reply:
updated